.gitlab-ci.yml

```diff
@@ -197,6 +197,15 @@ benchmarks-test:
   # we may live with failing benchmarks, it is just a signal for us
   allow_failure:                   true
 
+partial-repo-build-test:
+  stage:                           test
+  <<:                              *docker-env
+  <<:                              *nightly-test
+  script:
+    - ./scripts/verify-partial-repo-build.sh --no-revert
```

Comment:
@svyatonik 
very very nit, I would maybe name it like `verify-partial-repo-of-bridge-pallets-build.sh` or `verify-partial-bridge-pallets-build.sh` or something similar, because `partial-repo` could be relayer part or pallets part :)